examples/idea-examples/imdb/build.gradle.kts

```diff
@@ -0,0 +1,59 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    application
+    kotlin("jvm")
+    id("org.jetbrains.kotlinx.dataframe")
+}
+
+repositories {
+    mavenLocal() // in case of local dataframe development
+    mavenCentral()
+}
+
+kotlin.sourceSets.getByName("main").kotlin.srcDir("build/generated/ksp/main/kotlin/")
```

Comment:
Does it resolve & compile without this line?

Reply:
Better to create an issue for all examples (it's the same in all)